packages/core/aqua-to-js/src/generate/header.ts

```diff
@@ -35,12 +37,15 @@ export default function generateHeader(
  */
 ${
   outputType === "ts"
-    ? "import type { IFluenceClient as IFluenceClient$$, CallParams as CallParams$$ } from '@fluencelabs/js-client';"
+    ? "import type { IFluenceClient as IFluenceClient$$, ParticleContext as ParticleContext$$ } from '@fluencelabs/js-client';"
     : ""
 }
 
 import {
     v5_callFunction as callFunction$$,
     v5_registerService as registerService$$,
-} from '@fluencelabs/js-client';`;
+    FluencePeer as FluencePeer$$
+} from '@fluencelabs/js-client';
+
+${converters}`;
```

Comment:
I don't think this is a good idea to hardcode the body of these functions in each compiled file
Don't understand the reason to do this
You already export `v5_callFunction`, `v5_registerService` and `FluencePeer` from `'@fluencelabs/js-client'` why not export these functions as well from `'@fluencelabs/js-client'`  and use them normally

Reply:
It went through hard refactoring process and end up in the `aqua-to-js` as i decided to completely remove any schema mentions from js-client. It uses schema to perform conversion. We have 3 options
1. Make an exception for this case (easier)
2. Minify and uglify this snippet to minimize it's impact on frontend (every function from every file would be bundled here)
3. Support batch compilation in `aqua-to-js` package. As a result every batch compilation will generate a single `converter.js` file

Reply:
Decided to move it back to js-client